armi/tests/detailedAxialExpansion/refSmallReactorBase.yaml

```diff
@@ -461,80 +516,51 @@ blocks:
-        height: *highOffset_height
-        axial mesh points: *standard_axial_mesh_points
-        material modifications:
-            U235_wt_frac: &lta_fuel_u235_wt_frac ['', '', 0.2, 0.2, 0.2, '', '', '', '', '']
-            ZR_wt_frac: &lta_fuel_zr_wt_frac ['', '', 0.07, 0.07, 0.06, '', '', '', '', '']
-        xs types: *igniter_fuel_xs_types
-    lead test fuel b:
-        specifier: LB
-        blocks: [*block_grid_plate, *block_fuel_axial_shield, *block_lta2_fuel, *block_lta2_fuel, *block_lta2_fuel, *block_plenum, *block_aclp, *block_plenum, *block_duct, *block_dummy]
+        blocks: [*block_grid_plate, *block_fuel_multiPin_axial_shield, *block_fuel_multiPin, *block_fuel_multiPin, *block_fuel_multiPin, *block_mixed_multiPin, *block_aclp_multiPin, *block_plenum_multiPin, *block_duct, *block_dummy]
         height: *highOffset_height
         axial mesh points: *standard_axial_mesh_points
         material modifications:
-            U235_wt_frac: *lta_fuel_u235_wt_frac
-            ZR_wt_frac: *lta_fuel_zr_wt_frac
-        xs types: *igniter_fuel_xs_types
-    feed fuel:
-        specifier: OC
-        blocks: *igniter_fuel_blocks
-        height: [25.0, 25.625, 25.625, 25.625, 25.625, 22.5, 25.0, 25.0, 25.0, 17.5]
-        axial mesh points: *standard_axial_mesh_points
-        material modifications:
-            U235_wt_frac: *igniter_fuel_u235_wt_frac
-            ZR_wt_frac: *igniter_fuel_zr_wt_frac
-        xs types: *igniter_fuel_xs_types
-    secondary control:
-        specifier: SC
-        blocks: [*block_grid_plate, *block_ctrl_duct, *block_ctrl_duct, *block_control, *block_control, *block_control, *block_control_plenum, *block_ctrl_duct, *block_ctrl_duct, *block_dummy]
-        height: [25.0, 49.0, 49.0, 25.0, 25.0, 25.0, 25.0, 1.0, 1.0, 17.5]
-        axial mesh points: *standard_axial_mesh_points
+            U235_wt_frac: ['', '', 0.2, 0.2, 0.2, 0.2, '', '', '', '']
+            ZR_wt_frac: ['', '', 0.07, 0.07, 0.07, 0.07, '', '', '', '']
         xs types: *igniter_fuel_xs_types
-    primary control:
+    control:
         specifier: PC
         blocks: [*block_grid_plate, *block_ctrl_duct, *block_ctrl_duct, *block_control, *block_control, *block_control, *block_control_plenum, *block_ctrl_duct, *block_ctrl_duct, *block_dummy]
-        height: [25.0, 1.0, 1.0, 25.0, 25.0, 25.0, 25.0, 49.0, 49.0, 17.5]
+        height: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
         axial mesh points: *standard_axial_mesh_points
         xs types: *igniter_fuel_xs_types
     radial shield:
         specifier: SH
         blocks: [*block_grid_plate, *block_radial_shield, *block_radial_shield, *block_radial_shield, *block_radial_shield, *block_shield_plenum, *block_shield_aclp, *block_shield_plenum, *block_duct, *block_dummy]
-        height: [25.0, 25.0, 25.0, 25.0, 25.0, 25.0, 25.0, 25.0, 25.0, 17.5]
+        height: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
```

Comment:
changing these block heights to 1.0 is new for this PR. otherwise it's the same as #2162